CORTEX_M0+_RP2040/UsingCMSIS/main.c

```diff
@@ -79,7 +77,7 @@ int main(void) {
 }
 /*-----------------------------------------------------------*/
 
-static void prvBlinkTask( void *pvParameters )
```

Comment:
My compiler errored because these parameters are unused.

Reply:
Each FreeRTOS task function is required to have this function signature. A better way to address this would be to add the following line to the function definition:
```c
( void ) pvParameters;
```

Reply:
Got it. Thanks for the feedback. Fixed.